### ISSUE TITLE
error beego.Error()

### ISSUE THREAD

**Issue by YoungsoonLee** [astaxie/beego#3588]
Please answer these questions before submitting your issue. Thanks!

1. What version of Go and beego are you using (`bee version`)?
     go version go1.12.1 linux/amd64
    bee v1.10.0

2. What operating system and processor architecture are you using (`go env`)?
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/ubuntu/.cache/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/ubuntu/work"
GOPROXY=""
GORACE=""
GOROOT="/snap/go/3417"
GOTMPDIR=""
GOTOOLDIR="/snap/go/3417/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build601098611=/tmp/go-build -gno-record-gcc-switches"

----------------------------------------------------------------------------------------------------------------

  I deployed my project to live server and when I start bee run on Ubuntu,  I got the error like that. 
          undefined: beego.Error
   I wrote it in my code for logging beego.Error. But I got the error.

   But When I run bee  on Mac my laptop, It's okay,
   I think If beego install new one, get the error.

   Could you let me know anything to solve that?

**Comment by YoungsoonLee** [comment]
I think If beego install new one, get the error.

**Comment by jotavargas** [comment]
we have it same error, in pipeline with drone CI
temporary solution:
 cd $GOPATH/src/github.com/astaxie/beego
 git checkout 422e8285b566c21bd8b01b30f52e2aafbad1c4fa

**Comment by wwts17** [comment]
`/log.go` was removed, you should call functions in  ` github.com/astaxie/beego/logs`  package

**Comment by hurisheng** [comment]
Our code uses `beego.Error()` function for logging, but we found there will be an compilation error when we compile the code in a fresh environment in which we install beego by `go get github.com/astaxie/beego`.

After looking at the source code of beego, we found the `log.go` was deleted in develop branch. That's the reason we got `undefined: beego.Error`. 

1. Shouldn't the default branch remains `master`? That branch is supposed to be the stable version of beego. `develop` branch might contains some changes without further tests. The default behavior of `go get` retrieves the default version of repository according to the document:
```
When checking out or updating a package, get looks for a branch or tag that matches the locally installed version of Go. The most important rule is that if the local installation is running version "go1", get searches for a branch or tag named "go1". If no such version exists it retrieves the default branch of the package.
```

2. Why this incompatible change is accepted? I am pretty sure we are not the only person who had this issue.
